packages/faucet/src/tx.ts

```diff
@@ -96,19 +67,20 @@ export class TransactionResult {
 
   constructor(pe: PromiEvent<any>) {
     pe.on('transactionHash', (hash: string) => {
-      debug('hash: %s', hash)
+      console.info('@TransactionResult hash: ', hash)
       this.hashFuture.resolve(hash)
     })
       .on('receipt', (receipt: TransactionReceipt) => {
-        debugTxObjects('receipt: %O', receipt)
+        console.info('@TransactionResult receipt: ', receipt)
         this.receiptFuture.resolve(receipt)
       })
       .on('error', ((error: any, receipt: TransactionReceipt | false) => {
+        console.info('got a future error', error, receipt)
```

Comment:
This error message can probably be reworded. `'@TransactionResult error:` to conform with the other ones? And maybe only log the receipt in the second branch of the `if` below.